lib/rubocop/cop/rspec/empty_hook.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      #  Checks for empty before and after hooks.
+      #
+      # @example
+      #   # bad
+      #   before {}
+      #   after do; end
+      #   before(:all) do
+      #   end
+      #   all(:all) { nil }
```

Comment:
Sorry, I'm not familiar with this construct. Where does it come from?

Reply:
I meant `after(:all)` :) will change